packages/cms/store/resource.ts

```diff
@@ -81,13 +87,18 @@ interface ResourceMethods extends State {
   ) => Promise<[AxiosResponse | null, Error | null]>
 
   applyFilter: (filter: ActiveFilter) => void
+
   clearFilter: (filter: ActiveFilter) => void
 }
 
 export const useResourceStore = create<ResourceState & ResourceMethods>(
   devtools((set, get) => ({
     filters: [],
 
+    async createResource(resource: ResourceContract, resourceInput: AbstractData) {
```

Comment:
This method does not need resource as an argument. The resource is already in the store. You can get it like this:

```
const { resource } = get()
```